packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts

```diff
@@ -288,7 +288,7 @@ export class V8 {
       .catch(cb);
   }
 
-  getTransactions(wallet, startBlock, cb) {
+  getTransactions(wallet, startBlock, cb, includeInvalidTxs = false) {
```

Comment:
Let's keep the callback as the last param

Reply:
Updated